salt/utils/cloud.py

```diff
@@ -124,6 +124,9 @@
     r"(?:.*sudo)(?:.*)[Pp]assword(?: for .*)?:", re.M
 )
 
+SERVER_ALIVE_INTERVAL = 10
```

Comment:
I think 10 seconds is a very short period to wait before sending a null packet to the server, 30 or 60 would make more sense